common/logging/tests/test.rs

```diff
@@ -0,0 +1,57 @@
+#[cfg(test)]
+mod tests {
```

Comment:
As this is an integration test (lives in `tests/`) the `cfg(test)` and `mod tests` are both redundant

Reply:
I mean, you can just remove the level of nesting and just have `#[test]`